### ISSUE TITLE
I can`t install azk on ubuntu

### ISSUE THREAD

**Issue by nuxman** [azukiapp/azk#655]
Hi Today I  tried to install azk but all the times failed ! 
I think to have  a problem when  getting a key on keyserver, just look below 
I typed this 
``
$   wget -nv http://azk.io/install.sh -O- -t 2 -T 10 | sh
``
<pre>
[azk] Installing azk..........................................[ WAIT ]
[azk]   apt-key adv --keyserver keys.gnupg.net --recv-keys 022856F6D78159DF43B487D5C82CF0628592D2C9
[azk] Installing azk..........................................[ FAIL ]

[azk] [ERROR] Failed to install azk. Try again later.
[azk] [ERROR] Failed to install azk.
</pre>
a. Which OS / distro? UBUNTU 16.04 
My docker -v
Docker version 1.11.0-rc5, build 6178547

When I tried  to  type this on terminal, I got some errors  
<pre>
sudo apt-key adv --keyserver keys.gnupg.net --recv-keys 022856F6D78159DF43B487D5C82CF0628592D2C9 
Executing: /tmp/tmp.O1Nw3Cr7Cp/gpg.1.sh --keyserver
keys.gnupg.net
--recv-keys
022856F6D78159DF43B487D5C82CF0628592D2C9
gpg: requesting key 8592D2C9 from hkp server keys.gnupg.net
?: [fd 4]: read error: Connection reset by peer
gpgkeys: HTTP fetch error 7: couldn't connect: eof
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
gpg: keyserver communications error: keyserver unreachable
gpg: keyserver communications error: public key not found
gpg: keyserver receive failed: public key not found
</pre>
I appreciate some help ... cheers

**Comment by fearenales** [comment]
Hi @nuxman!

Thank you for your feedback.

Even `azk` not being supported for Ubuntu 16.04 yet, the error is occurring during the GPG key fetch. May you try fetching it from another key server?

You can do this by running:

```
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 022856F6D78159DF43B487D5C82CF0628592D2C9 
```

If that works, you'll need to build `azk` from the source, since we still don't have an `azk` package for Ubuntu 16.04. You can find a tutorial for doing this in [our docs](http://docs.azk.io/en/installation/source-code.html).

**tl;dr**:

1) Install `libnss-resolver` from source following [these instructions](https://github.com/azukiapp/libnss-resolver#from-the-source-without-azk);
3) `git clone https://github.com/azukiapp/azk.git` (remember to navigate to a folder where you want `azk` to be cloned);
4) `cd azk && git checkout stable`;
5) `make`;
6) Add the `/your/path/to/azk/bin/azk` into you `$PATH` env var;

You can test it by running `azk version --full`.

Any issue you may have, please let us know.

**Comment by nuxman** [comment]
cheer I will do this ... see ya .. if I fix I will post here salute !!!

**Comment by nuxman** [comment]
Hi @fearenales  
my azk version 
➜  ~  azk version --full 
Version   : azk version 0.18.0, build 109e8ac, date 2016-04-07
OS        : Linux 4.4 (x64), Memory: 15959MB
Agent     : Stopped
Docker    : 1.11.0 (CLI)
Uses VM   : No
VirtualBox: N/A

But the key didn't work ... thanks for you help

**Comment by fearenales** [comment]
Hi @nuxman !

Good to know you made it!

I'll dig the GPG key issue a little deeper. That's why I'll let this issue open.

Also, I'll open a new one for tracking Ubuntu 16.04 support.

**Comment by fearenales** [comment]
I'm closing this issue since Ubuntu 16.04 is already supported by `azk`.

To install `azk` on Ubuntu (12.04/14.04/15.10/16.04), simply run:

```
curl -sSL azk.io/install.sh | sh
```
